### PR TITLE
Solidstart feature: gists fetch

### PR DESCRIPTION
**Pull request by Megio**
Add query and fetch function for retrieve gists data

closes: #1351

### CODE REVIEW

**Review by AllStackDev1 — DISMISSED** [review]
LGTM

File: solidjs-tailwind/src/services/get-gists.js [hunk 1]
```diff
@@ -24,14 +24,14 @@ const getGists = async () => {
       (_acc, file) =>
```
Comment (SDaian) [inline solidjs-tailwind/src/services/get-gists.js:24]: we shouldn't modify other Pod's file, please revert this change

**Review by SDaian — CHANGES_REQUESTED** [review]

File: solidjs-tailwind/src/services/get-gists.js (hunk 1 above)
Comment (hdJerry) [reply solidjs-tailwind/src/services/get-gists.js:24]: This is a result of the husky not being correctly configured. But has been now. Kindly revert, please. THanks sir

File: solidstart-tanstackquery-tailwind-modules/src/services/api.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export type ApiProps = {
   url: string;
   query: string | null;
-  variables: Record<string, string | number | null> | null;
+  variables?: Record<string, string | number | null> | null;
```
Comment (hdJerry) [inline solidstart-tanstackquery-tailwind-modules/src/services/api.ts:4]: This will likely conflict with some of your recent changes

**Review by SDaian — APPROVED** [review]
LGTM
